stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -129,10 +130,6 @@ public class ChatClient internal constructor(
     private val userStateService: UserStateService = UserStateService(),
     private val tokenUtils: TokenUtils = TokenUtils,
     internal val scope: CoroutineScope,
-    private val appContext: Context,
```

Comment:
Really good we are removeing `Context` dependency on our `ChatClient`!!!! :partying_face: :tada:

Reply:
🎉 🎉 🎉